src/front/wgsl/mod.rs

```diff
@@ -1257,6 +1257,72 @@ impl Parser {
             }
         } else {
             match name {
+                "bitcast" => {
+                    let _ = lexer.next();
+                    lexer.expect_generic_paren('<')?;
+                    let ((ty, _access), type_span) = lexer.capture_span(|lexer| {
+                        self.parse_type_decl(lexer, None, ctx.types, ctx.constants)
+                    })?;
+                    lexer.expect_generic_paren('>')?;
+
+                    let expr = self.parse_general_expression(lexer, ctx.reborrow())?;
```

Comment:
You need to call `lexer.{open,close}_arguments()` otherwise you'll allow code like
```rust
var a = bitcast<f32>1;
```